doc/site/articles/yardmaps.markdown

```diff
@@ -52,10 +52,12 @@ Basic characters:
 | u      | ❌                    | ✅                   | ❌                      | ✅                     | ❌          | ❌         |
 | e      | ❌                    | ⚠️ exit only         | ❌                      | ⚠️ exit only           | ❌          | ❌         |
 
-Others:
+Remarks:
 
- * `h` has a special meaning - it **has to be the first letter and means a high resolution yardmap**,
-i.e. there have to be 4x more characters and each will correspond to a "regular" 8x8 elmo game square instead of the larger 16x16 elmo footprint square.
+ * Needs geo means the unit can only be built on top of terrain marked as geothermal.
+ * `h`: special character that when passed as (and **only valid** when) the first letter on a yardmap (e.g. `hyyy..`) marks it as high resolution: requires 4x more characters and each will correspond to a "regular" 8x8 elmo game square instead of the larger 16x16 elmo footprint square. Does not count as a square.
+ * `c`: Open and unblocking only while constructing (controlled by animation via `COB.YARD_OPEN`)
```

Comment:
Thought relevant to include

Reply:
Yes, `COB.YARD_OPEN` is what the table means by active/inactive, and it isn't actually related to constructing. [For example solars opening/closing use it](https://github.com/beyond-all-reason/Beyond-All-Reason/blob/a0565ce1484df67f04a5e002cf5b7cdeae05b778/scripts/Units/armsolar.bos#L71)

Reply:
Thank you, that clarifies/simplifies things.